ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -85,6 +95,259 @@ const triggerSet: TriggerSet<Data> = {
+          return;
+
+        const poly6Map: { [l: string]: string } = {
+          insideNW: 'insideSE',
+          insideNE: 'insideSW',
+          insideSE: 'insideNW',
+          insideSW: 'insideNE',
+        };
+
+        switch (data.polyInstance) {
+          case 1: // four safe spots: two inside, two outside.  call only the inside ones.
+            if (safeTiles.length !== 4 || safe1 === undefined)
+              return;
+            return output.combo!({ dir1: output[safe0]!(), dir2: output[safe1]!() });
+          case 2: // one inside safe spot
+            if (safeTiles.length !== 1)
+              return;
+            return output.single!({ dir1: output[safe0]!() });
+          case 3: // two inside safe spots
+            if (safeTiles.length !== 2 || safe1 === undefined)
+              return;
+            return output.combo!({ dir1: output[safe0]!(), dir2: output[safe1]!() });
+          case 4: // lots of safe spots, so give generic warning
+            return output.polyAvoid!();
+          case 5: // two outside safe spots (reduced to one by Chorus Ixou)
+            if (safeTiles.length !== 2 || safe1 === undefined)
+              return;
+            // TODO: Maybe call only a single tile, and only once Chorus Ixou direction is known?
+            return output.combo!({ dir1: output[safe0]!(), dir2: output[safe1]!() });
+          case 6: // Cachexia 2 - four safe spots that form corners of a 3x3 tile sub-grid, so just call the center
+            if (safeTiles.length !== 4)
+              return;
+            if (poly6Map[safe0] === undefined)
+              return;
+            return output.polyCachexia!({ dir1: output[poly6Map[safe0]!]!() });
+          case 7: // one inside safe spot
+            if (safeTiles.length !== 1)
+              return;
+            return output.single!({ dir1: output[safe0]!() });
+          case 8: // four safe spots, two inside two outside.  call only the inside ones.
+            if (safeTiles.length !== 4 || safe1 === undefined)
+              return;
+            return output.combo!({ dir1: output[safe0]!(), dir2: output[safe1]!() });
+          default:
+            return;
+        }
+      },
+      run: (data) => {
+        data.tileTethers = [];
+        data.mapEffects = [];
+        data.combatantData = [];
+      },
+      outputStrings: {
+        combo: {
+          en: 'Safe Tiles: ${dir1} / ${dir2}',
+        },
+        single: {
+          en: 'Safe Tile: ${dir1}',
+        },
+        polyCachexia: {
+          en: 'Safe Tiles: Intercards of ${dir1}',
+        },
+        polyAvoid: {
+          en: 'Avoid Unsafe Tiles',
+        },
+        insideNW: {
+          en: 'Inside NW',
+        },
+        insideNE: {
+          en: 'Inside NE',
+        },
+        insideSE: {
+          en: 'Inside SE',
+        },
+        insideSW: {
+          en: 'Inside SW',
+        },
+        // Corner tiles will never be safe for any version of Poly,
```

Comment:
This comment is incorrect, since poly 6/cache2 has safe corner spots. It would be more accurate to say that the trigger never calls out corner safe spots?

Reply:
Agreed - will update that too.